src/QPTasks.h

```diff
@@ -610,7 +610,7 @@ class TASKS_DLLAPI TransformTaskCommon : public HighLevelTask
 	{
 	}
 
-	tasks::TransformTask& task()
+  transform_task_t& task()
```

Comment:
Ok, I missed that the project is tab-based...
Yes, there was a mistake here. I guess it means the method is never used since never instantiated, or only with tasks::TransformTask